### PR TITLE
feat: update to the latest version of Emotion

### PR DESCRIPTION
**Pull request by Jared-Dev**
Code changes were limited to what was required to work
with the library upgrades.

I am getting an error related to `wasm` (I believe). 

```
Using browser-only version of superagent in non-browser environment
error UNHANDLED REJECTION


  Error: You must provide the URL of lib/mappings.wasm by calling SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using SourceMapConsumer

  - read-wasm.js:8 readWasm
    [store.gatsbyjs.org]/[react-hot-loader]/[source-map]/lib/read-wasm.js:8:13

  - wasm.js:25 wasm
    [store.gatsbyjs.org]/[react-hot-loader]/[source-map]/lib/wasm.js:25:16

  - source-map-consumer.js:264 BasicSourceMapConsumer.then.that
    [store.gatsbyjs.org]/[react-hot-loader]/[source-map]/lib/source-map-consumer.js:264:14
```

Sending the PR to see if someone can point me in the right direction to getting those resolved.

### CODE REVIEW

**Comment by m-allanson** [comment]
Thanks @Jared-Dev! Can you try updating Gatsby to version `2.3.34`. There was a regression that affected some sites in version 2.3.31.

File: src/components/Cart/Cart.js [hunk 1]
```diff
@@ -25,7 +26,7 @@ import {
   dimensions
 } from '../../utils/styles';
 
-const CartRoot = styled(`div`)`
+const CartRoot = styled.div`
```
Comment (jlengstorf) [inline src/components/Cart/Cart.js]: Can you revert these to the ```styled(`div`)``` syntax, please? Our internal code style prefers that to keep consistency with custom elements (e.g. ```styled(MyComponent)```).

File: src/components/Layout/Layout.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
+/** @jsx jsx */
```
Comment (jlengstorf) [inline src/components/Layout/Layout.js]: `gatsby-plugin-emotion` adds the babel plugin to process the CSS prop, so this pragma change isn't necessary.

File: src/components/Layout/Logo.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
+/** @jsx jsx */
```
Comment (jlengstorf) [inline src/components/Layout/Logo.js]: No need for the pragma.

File: src/components/shared/SiteMetadata.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
+/** @jsx jsx */
```
Comment (jlengstorf) [inline src/components/shared/SiteMetadata.js]: No pragma.

**Comment by Jared-Dev** [comment]
I'll get rid of the pragma everywhere. I switched the syntax so that highlighting worked everywhere, but I can switch that back.

**Comment by jlengstorf** [comment]
@Jared-Dev Thanks!

**Comment by Jared-Dev** [comment]
@m-allanson & @jlengstorf this should be fixed now. I clicked around the site and didn't see any errors.

**Pull request merged**